index.html

```diff
@@ -560,17 +560,59 @@ <h4>$schema</h4>
                </p>
             </section>
             <section class="normative">
-               <h4>name</h4>
+               <h4><dfn>title</dfn></h4>
                <p>
-                  It is RECOMMENDED that all JSON Schemas include a <code>name</code> property which provides
-                  a human-readable name that describes the schema.
+                  It is RECOMMENDED that all JSON Schemas include the optional <code>title</code> property as defined in
+                  [[JSON-SCHEMA]].
                </p>
             </section>
             <section class="normative">
-               <h4>description</h4>
+               <h4><dfn>description</dfn></h4>
                <p>
-                  JSON Schemas MAY choose to include an optional <code>description</code> property which provides
-                  a human-readable sentence describing the schema.
+                  It is RECOMMENDED that all JSON Schemas include the optional <code>description</code> property as
+                  defined in [[JSON-SCHEMA]].
+               </p>
+            </section>
+            <section class="normative">
+               <h4>dir</h4>
+               <p>
+                  JSON Schemas MAY choose to include an optional <code>dir</code> property that indicates the
+                  base direction for the values of the [=title=] and [=description=] properties defined in
```

Comment:
consider linking "base direction" to the I18N glossary

```suggestion
[=base direction=] for the values of the [=title=] and [=description=] properties defined in
```

(instructions on how to add the i18n-glossary to your spec are [here](https://w3c.github.io/i18n-glossary/#respec-how-to-use))
```